### ISSUE TITLE
Icon List Element

### ISSUE THREAD

**Issue by Netzlichter** [soflyy/oxygen-bugs-and-features#1291]
**Describe the feature you'd like to see included in Oxygen.**
An Icon List Element would be awesome.

**What are the use cases for this feature?**
To easely make lists with icons in front

**Examples of this feature or functionality.**
https://docs.elementor.com/article/50-icon-list

:)!

**Comment by steve-goldberg** [comment]
A simple way would to be use the Icon Box element, but have a field to display the title or text, so it's just an Icon and text. But this way wouldn't add the ul and li classes.

**Comment by Netzlichter** [comment]
Yes. This is like I do it at the moment but it is much less comfortable and like you mentioned, list should have ul, li classes...
